doc/whats_new/v1.3.rst

```diff
@@ -96,6 +96,13 @@ Changelog
 - |Feature| A `__sklearn_clone__` protocol is now available to override the
   default behavior of :func:`base.clone`. :pr:`24568` by `Thomas Fan`_.
 
+:mod:`sklearn.calibration`
```

Comment:
I would move this in 1.2.2 because this is an unfortunate regression.

Reply:
Done